docs/rules/no-missing-import.md

```diff
@@ -69,13 +69,30 @@ If a path is relative, it will be resolved from CWD.
 
 Default is `[]`
 
+#### typescriptExtensionMap
```

Comment:
To avoid duplication, I'm just wondering whether we could add a section to the readme that describes the shared settings. In the rule docs we can add a link to that. (surely not blocking merging this PR. 😄 )